tests/queries/0_stateless/02703_row_policies_database_combination.sql

```diff
@@ -0,0 +1,88 @@
+-- Tags: no-parallel
```

Comment:
Let's rename this file too to `02703_row_policies_for_database_combination.sql`

Reply:
Sure, I've just missed it.